src/main/target/common_post.h

```diff
@@ -552,14 +552,6 @@ extern uint8_t __config_end;
 #define USE_CUSTOM_DEFAULTS_ADDRESS
 #endif
 
-#if defined(USE_RX_EXPRESSLRS) && defined(STM32F411)
-#define RX_SPI_DEFAULT_PROTOCOL          RX_SPI_EXPRESSLRS
-#endif
-
-#if defined(USE_RX_EXPRESSLRS) && !defined(RX_EXPRESSLRS_TIMER_INSTANCE) && (defined(STM32F411) || defined(STM32F405) || defined(STM32F745) || defined(STM32H7))
-#define RX_EXPRESSLRS_TIMER_INSTANCE     TIM5
```

Comment:
have the corresponding targets that were relying on this been also updated to include the appropriate #define in the unified targets repo and in the `src/config`s folder.